cuqi/experimental/mcmc/_rto.py

```diff
@@ -228,6 +246,8 @@ def _initialize(self):
             self.solver = "FISTA" if callable(self.proximal) else "ADMM"
         if self.solver == "FISTA":
             self._stepsize = self._choose_stepsize()
+        self.compute_map_regularized()
+        self.initial_point_per_step = None
```

Comment:
I also suggest these two to be hidden
```suggestion
        self._compute_map_regularized()
        self._initial_point_per_step = None
```